QC/wdl/workflows/snv_indel_assembly.wdl

```diff
@@ -0,0 +1,190 @@
+version 1.0
+
+import "../tasks/deepvariant.wdl" as runDeepVariant
+import "../tasks/pepperMarginDeepVariant.wdl" as runPepperMarginDeepVariant
+import "../tasks/hapDotPy.wdl" as runHappy
+
+
+workflow snv_indel_assembly {
+
+    meta {
+        author: "Julian Lucas"
+        email: "juklucas@ucsc.edu"
+        description: "WDL implementation of the small variant calling section of the [T2T Polishing Case Study](https://github.com/arangrhie/T2T-Polish/blob/master/doc/T2T_polishing_case_study.md)."
+    }
+
+    input {
+        File assembly
+        File assemblyIndex
+        String sample
+    }
+
+    ## Call DeepVariant (recommended input of HiFi + Ilmn data)
+    call runDeepVariant.DeepVariant as deepVariant_t {
+        input:
+            assembly      = assembly,
+            assemblyIndex = assemblyIndex,
+            sample        = sample
+    }
+
+    ## Call PepperMarginDeepVariant (ONT data)
+    call runPepperMarginDeepVariant.pepperMarginDeepVariant as pmdv_t {
+        input:
+            assembly      = assembly,
+            assemblyIndex = assemblyIndex,
+            sample        = sample
+    }    
+
+    ## Filter DeepVariant calls
+    call runPepperMarginDeepVariant.bcftoolsFilter as filt_1 {
```

Comment:
Maybe more informative call names than filt_1 and filt_2? But I am nitpicking at this point!

Reply:
Agreed. I cringed when I wrote this, but pushed through for expediency. I'll make the change. Thx.